src/mmw/js/src/modeling/tests.js

```diff
@@ -90,6 +90,14 @@ describe('Modeling', function() {
                 assert.equal($('#sandbox ul.nav.nav-tabs > li').length, 3);
             });
 
+            function checkItemsMatch(expectedItems) {
```

Comment:
Since this function is not generic and only matches menu items, consider renaming to `checkMenuItemsMatch`